ansible/roles/hana_checks/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: HANA check - Wait pacemaker up
+  become: true
+  command: systemctl --no-pager is-active pacemaker
+  register: pm
+  retries: "{{ pacemaker_timeout // 15 }}"
+  delay: 15
+  until: pm.stdout == 'active'
+
+- name: HANA check - Wait cluster idle again
+  become: true
+  command: cs_wait_for_idle --sleep 5
+  register: idle2
+  retries: "{{ cs_wait_timeout // 5 }}"
+  delay: 5
+  until: idle2.rc == 0
+
+- name: HANA check - Gather topology JSON
+  become: true
+  command: SAPHanaSR-showAttr --format=script
```

Comment:
... but here you are using `script` format.

I know not all the versions of `SAPHanaSR-showAttr` are supporting json format but it will be interesting to have a task using it, maybe side by side to the one using `script`

Reply:
It's not portable, but we can pass `use_angi` to ansible as a variable. I already have a part of the code in `secondary` that uses it optionally (if it exists):

```
master_resource_type: "{{ use_angi | default(false) | bool | ternary('mst','msl') }}"
```